### DISCUSSION TITLE
Content for 2023

### DISCUSSION THREAD

**Discussion by clearcoding** [aws-lug/aws-lug-meetups#29] [Ideas]
Some possible options for 2023 speakers
Dave (IW)  - Data lakes and kinesis pipes
AWS - Proton and Backstage

**Comment by clearcoding** [comment]
Simon (IW)  - Build it/ Run it from experience

**Comment by clearcoding** [comment]
Saturday session 22nd April consider a workshop type meeting0

**Comment by clearcoding** [comment]
Other ideas
Panel session either with people trading experiences or comparing solutions/technologies
Mini hackathon for social themed problems

**Reply by clearcoding** [reply]
@pmgledhill102 is going to chase AWS for this session

**Comment by clearcoding** [comment]
The sustainability in the cloud session. We have connections to local based people exploring topics in this field

**Comment by pmgledhill102** [comment]
Matt Houghton (https://www.youtube.com/watch?v=VWfvD_9J6z8)... did a talk at ComSum and Re:invent -> and could do a talk
